python/tutorials/03-matrix-multiplication.py

```diff
@@ -312,6 +312,24 @@ def matmul(a, b, activation=""):
 else:
     print("❌ Triton and Torch differ")
 
+
+TORCH_HAS_FP8 = hasattr(torch, "float8_e5m2")
+if TORCH_HAS_FP8:
+    torch.manual_seed(0)
+    a = torch.randn((512, 512), device="cuda", dtype=torch.float16)
+    b = torch.randn((512, 512), device="cuda", dtype=torch.float16)
+    a = a.to(torch.float8_e5m2)
+    b = b.to(torch.float8_e5m2)
+    triton_output = matmul(a, b)
+    torch_output = torch.matmul(a.to(torch.float16), b.to(torch.float16))
+    print(f"triton_output={triton_output}")
```

Comment:
we should probably add some printing to tell that this is fp8 inputs otherwise the output is confusing.

Reply:
added fp16 and fp8 suffix to indicate the type of the output. thanks